src/org/exist/security/internal/SecurityManagerImpl.java

```diff
@@ -96,14 +97,14 @@
     private final PrincipalLocks<Account> accountLocks = new PrincipalLocks<>();
     private final PrincipalLocks<Group> groupLocks = new PrincipalLocks<>();
 
-    //TODO: validate & remove if session timeout
     private SessionDb sessions = new SessionDb();
 
-    @ConfigurationFieldAsAttribute("last-account-id")
-    protected int lastUserId = 0;
+    public final static int INITIAL_LAST_ACCOUNT_ID = 10;
+    public final static int INITIAL_LAST_GROUP_ID = 10;
 
-    @ConfigurationFieldAsAttribute("last-group-id")
-    protected int lastGroupId = 0;
+    //calculated runtime
+    protected int lastAccountId = INITIAL_LAST_ACCOUNT_ID;
```

Comment:
It seems to me that this can only be safely accessed with a lambda to `accountsById#modify` to ensure consistency. The current design breaks the encapsulation of the concurrency concerns, perhaps it would be better to move the `lastId` field into `PrincipalDbById` or to flatten out `PrincipalDbById` into `SecurityManagerImpl`?